C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -0,0 +1,20 @@
+#include <bits/stdc++.h>
+using namespace std;
+int F(int n){
+    int arr[n+1],i,j;
```

Comment:
It looks like you have copy pasted the code from somewhere.  Follow proper indentation and refer to [C++ style guide](http://users.csc.calpoly.edu/~cstaley/General/CStyle.htm) to improve readability of your code. Make sure you go through all the 12 points mentioned in #398 .

Reply:
No, sir actually I have been in competitve programming for more than a year. I write much shorter codes than this. I have actually written longer in comparison to code I write usually.Sir, @atarax665 please advice me, should I make it more clear like mentioning opening and closing parenthesis for loops. Sir, actually I have written wrong statement in issue. F(n) denotes the number of coprimes of n from 1 to n. Example F(4) denotes number of coprimes of 4 from 1 to 4, so F(4)=2(as only 1 & 3 are coprimes to 4).

Reply:
@atarax665 Sir, please advice me what should I do now.

Reply:
Competitive coding really impacts our code writing habits, I feel you.
 No problem do the following changes and you will be good to merge:
- Add  parenthesis wherever required following proper indentation.
- Add the description of the problem in the beginning with complexity.
- Add sample input/output for your code in the end of your program
- Give space before and after operators to make the code more readable.